org.mixedrealitytoolkit.core/Utilities/Extensions/InteractorHandednessExtensions.cs

```diff
@@ -38,5 +38,26 @@ public static XRNode ToXRNode(this InteractorHandedness hand, XRNode defaultValu
                     return defaultValue;
             }
         }
+
+        /// <summary>
+        /// Converts the <see cref="InteractorHandedness"/> to <see cref="Handedness"/>. If the <see cref="InteractorHandedness"/>
+        /// is other than InteractorHandedness.Left or InteractorHandedness.Right then it defaults to <see cref="Handedness"/>.None.
+        /// </summary>
+        /// <param name="hand">The <see cref="InteractorHandedness"/> value for
+        /// which the <see cref="Handedness"/> is requested.</param>
+        /// <see cref="Handedness"/> representing the specified <see cref="InteractorHandedness"/>.
+        /// <returns></returns>
+        public static Handedness ToMRTKHandedness(this InteractorHandedness hand)
```

Comment:
Please remove "MRTK" in the name.

Reply:
Done in https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/pull/750/commits/5240490571364031de957fbfa047de332070e8cc , resolving.